aws-datastore/src/main/java/com/amplifyframework/datastore/DataStoreConflictHandlerResult.java

```diff
@@ -25,17 +25,17 @@
      * Conflict is handled by discarding the local (client-side) changes, preferring whatever
      * was on the server.
      */
-    DISCARD,
+    APPLY_REMOTE,
 
     /**
      * Conflict is handled by keeping the local (client-side) changes, discarding whatever
      * was on the server.
```

Comment:
Thinking about this description and its precision. The conflict will be handled by retrying the local change which attempts to persist it again on the backend. Is it a given that the end result "discarding whatever was on the server" will occur, or could it result in another conflict (say, from another client)?

Reply:
I'm going to update the wording here. Will also remove the discarding part because the retry may actually result in another conflict